common/d_flip_flop/t_flip_flop.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component version="1.2" xmlns="http://schemas.circuit-diagram.org/circuitDiagramDocument/2012/component/xml">
+  <declaration>
+    <meta name="name" value="T Flip-Flop" />
```

Comment:
All components need a unique ID. I've generated one here if you want to use this:

```suggestion
    <meta name="name" value="T Flip-Flop" />
    <meta name="guid" value="9a442c47-4909-4ced-a77e-9d2d9b244e7f" />
```